cfme/configure/access_control.py

```diff
@@ -1326,6 +1322,8 @@ class Tenant(Updateable, BaseEntity):
     parent_tenant = attr.ib(default=None)
     _default = attr.ib(default=False)
 
+    obj_type = 'Tenant'
```

Comment:
**Required**

Put this above L1320 please, class attributes before the object attributes.

Reply:
Done